### ISSUE TITLE
TileStache GeoJSON compatibility

### ISSUE THREAD

**Issue by radumas** [Leaflet/Leaflet.VectorGrid#106]
TileStache can serve slices of GeoJSON from PostGIS, presenting a URL of form:  
`http://example.org/path/tile.cgi/streets/12/656/1582.json`

The [example VectorGrid.Protobuf](https://leaflet.github.io/Leaflet.VectorGrid/vectorgrid-api-docs.html#vectorgrid-protobuf-example) URL is:  
`https://free-{s}.tilehosting.com/data/v3/{z}/{x}/{y}.pbf`. 

This seems like a match made to be no?

**Comment by perliedman** [comment]
Should most likely be very straight forward to add, given that GeoJSON already feeds data to the slicer, and we already load tiles in the protobuf implementation, as you already noted.

Having said that, I don't think it's likely that any of the maintainers will go ahead and just implement this, given pretty limited resources at the moment.

I'd be happy to answer questions to help you with this, if it's something you'd be willing to implement. Another option would be if you'd be willing to sponsor development of this feature.

**Comment by radumas** [comment]
I don't think this is as much a priority for me since [Tangram](https://github.com/tangrams/tangram#quickstart) works as a leaflet plugin and can parse either GeoJSON or protobuf vector tiles. I haven't actually tried it out yet
